OneWelcomeExampleApp/ViewControllers/Login/LoginViewController.swift

```diff
@@ -30,13 +30,13 @@ class LoginViewController: UIViewController {
 
     var profiles = [UserProfile]() {
         didSet {
-            reloadProfiles()
+            profilesTableView?.reloadData()
```

Comment:
I would keep the previous function's invocation here

Reply:
How about now?